change/@fluentui-react-migration-v8-v9-daf3e7a9-933f-46b2-8b57-d229db417da9.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Add colorNeutralBackgroundAlpha and colorNeutralStrokeAlpha tokens",
```

This is probably a feature addition no ?

```suggestion
  "comment": "feat: Add colorNeutralBackgroundAlpha and colorNeutralStrokeAlpha tokens",
```